composer.json

```diff
@@ -20,7 +20,7 @@
         "doctrine/inflector": "^2.0.9",
         "fidry/cpu-core-counter": "^1.1",
         "illuminate/container": "^10.43",
-        "nette/utils": "^3.2",
+        "nette/utils": "^4.0",
```

Comment:
@TomasVotruba it seems cause error on php 7.3:

```
"System error: "preg_replace_callback() expects at most 5 parameters, 6
         given" 
```

ref https://github.com/rectorphp/rector/actions/runs/8334283973/job/22807579480#step:5:12

Reply:
Resolved by https://github.com/rectorphp/rector-src/pull/5738 and https://github.com/rectorphp/rector-src/pull/5739

Reply:
Thank you 🙏